plugins/builds/index.js

```diff
@@ -267,17 +274,18 @@ async function createEvent(config) {
 }
 
 /**
- * Create external build
+ * Create external build (returns event with `builds` field)
  * @method createExternalBuild
  * @param  {Object}   config                    Configuration object
  * @param  {Factory}  config.pipelineFactory    Pipeline Factory
  * @param  {Factory}  config.eventFactory       Event Factory
- * @param  {String}   config.externalPipelineId External pipelineId
+ * @param  {Number}   config.externalPipelineId External pipeline ID
  * @param  {String}   config.startFrom          External trigger to start from
- * @param  {Number}   config.parentBuildId      Parent Build Id
+ * @param  {Number}   config.parentBuildId      Parent Build ID
  * @param  {Object}   config.parentBuilds       Builds that triggered this build
  * @param  {String}   config.causeMessage       Cause message of this event
- * @param  {Boolean}  [config.start]            Whether to start the build after creating
+ * @param  {Number}   [config.parentEventId]    Parent event ID
+ * @param  {Number}   [config.groupEventId]     Group parent event ID
  * @return {Promise}
  */
 async function createExternalBuild(config) {
```

Comment:
how about `createInternalBuild` function ? how is `groupEventId` handled there  ?

Reply:
`groupEventId` is a field for an `event`. `createExternalBuild` creates a new `event` (with subsequent builds), but `createInternalBuild` creates a single `build`, so there's no where to set the `groupEventId`.